tests/test_client.py

```diff
@@ -89,6 +89,17 @@ def test_list_engines(self):
             response = self.client.list_engines()
             self.assertEqual(response, expected_return)
 
+    def test_list_engines_with_pagination(self):
+        expected_return = [
+            { 'name': 'myawesomeengine' }
+        ]
+
+        with requests_mock.Mocker() as m:
+            url = "{}/{}".format(self.client.swiftype_session.base_url, 'engines')
+            m.register_uri('GET', url, json=expected_return, status_code=200)
+            response = self.client.list_engines(1, 20)
+            self.assertEqual(response, expected_return)
```

Comment:
this isn't actually the expected response - the expected response includes `meta` and `results` as top level keys